src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import profileReducer from './profile/profile';
```

Comment:
-  [ ] Please make sure to solve this warning 
![image](https://user-images.githubusercontent.com/74744787/195981655-a06ca36f-2fc0-4c9d-9849-ed61ab6fec64.png)
 
you can use this npm library 
![image](https://user-images.githubusercontent.com/74744787/195981686-4a6c6fd1-0672-46aa-918f-5a2821880e86.png)
you can use it like this 
![image](https://user-images.githubusercontent.com/74744787/195981735-98448ff0-2c48-4afb-bfdb-fd4114c96773.png)